src/Chirp.Core/DataTransferObjects/AuthorDTO.cs

```diff
@@ -0,0 +1,7 @@
+namespace Chirp.Core.DataTransferObjects;
+
+public class AuthorDTO
+{
+    public string Name { get; set; }

```

Comment:
specify Name as nullable, or add 'required' modifier (per github actions warning)

Reply:
This was following the convention we have created before